src/qt/choice.cpp

```diff
@@ -74,7 +74,7 @@ bool wxChoice::Create( wxWindow *parent, wxWindowID id,
         const wxValidator& validator,
         const wxString& name )
 {
-    return Create( parent, id, pos, size, choices.size(), &choices[ 0 ], style,
+    return Create( parent, id, pos, size, choices.size(), choices.size() ? &choices[ 0 ] : NULL, style,
```

Comment:
This should be `choices.empty() ? NULL : &choices[0]`.
`size()` will count the array elements and is slightly less efficient.